### PR TITLE
Adding the list of stages to build in the tutorial.

### PR DESCRIPTION
**Pull request by jonathanrocher**
Documenting the list of stages for us to build and go through with attendees.

### CODE REVIEW

**Review by siddhantwahal — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -60,6 +60,27 @@ During the tutorial, don't hesitate to ask for help:
   can be posted.
 
 
+## Outline of the tutorial
+The tutorial will take you through stages to go a from a script (stage 0) to 
+the packaged application. Exercises will get you to build the most significant 
+portions of the differences between each stages but each of them is provided as 
+a separate folder are provided to make sure everyone to get to the end point:
```
Comment (chamshop) [inline README.md:67]: Maybe this as an edit?
"The tutorial will guide you through all the stages from a basic python script (stage 0) to a fully packaged and installable application. The included exercises will walk you through developing the primary product of each stage. A solution is available for each exercise, though, to ensure all participants are able to reach the end goal."

File: README.md [hunk 2]
```diff
@@ -60,6 +60,27 @@ During the tutorial, don't hesitate to ask for help:
...
+
+  - step 0: python script
+  - step 1: traits-ified script
```
Comment (chamshop) [inline README.md:70]: Could we call step 1 "a more robust script with traits"? Everyone may not know what "traits" is at this point, so including what it does may help.

File: README.md [hunk 3]
```diff
@@ -60,6 +60,27 @@ During the tutorial, don't hesitate to ask for help:
...
+  - step 2: etstool, unit test and CI [OPTIONAL]
+  - step 3: traitsUI view for an image file
+      - File
+      - Enum
+      - Range
+      - readonly output
+      - Float
+  - step 4: pyface application: tree navigator and double click on an image to display the traitsUI view of the image.
+  - step 5: pyface application: add folder editor to displaying a table of metadata for all images inside. Add button to launch the face detection on all images. Add widgets to filter images.
```
Comment (chamshop) [inline README.md:79]: How about this as an edit?
```
step 5: pyface application
 - add folder editor to display a table of metadata for all images inside
 - add button to launch the face detection on all images
 - add widgets to filter images
```

**Review by chamshop — APPROVED** [review]
A few comments, otherwise thanks for putting this together!

File: README.md (hunk 1 above)
Comment (jonathanrocher) [reply README.md:67]: Thanks. Done.

File: README.md (hunk 2 above)
Comment (jonathanrocher) [reply README.md:70]: Good point, done.

File: README.md (hunk 3 above)
Comment (jonathanrocher) [reply README.md:79]: Thanks, done.

**Comment by jonathanrocher** [comment]
Thanks @chamshop @siddhantwahal . Comments addressed. Merging...

**Pull request merged**